rules/less-is-more/rule.md

```diff
@@ -0,0 +1,61 @@
+---
+type: rule
+title: Do you know that Less is more ?
+uri: less-is-more
+authors:
+  - title: Josep Fernandez
+    url: https://www.ssw.com.au/people/joseph-fernandez
+created: 2023-02-22T04:20:12.183Z
+guid: 9d8ce87b-f835-48a2-b60c-5549466da209
+---
+A popular concept in product development and interphase design is the idea of “Less is more”.  But what do we actually mean by less is more? 
+
+Let’s start off by defining what it doesn’t mean:
+
+❌ It doesn’t mean that you work less on a project and be able to deliver more
+
+❌ It doesn’t mean we make decisions based on our own personal biases or perceptions of what we think is the easiest solution for the user
+
+❌ It does not mean we Include or develop a feature in a design or product just because we have the technical ability to do so
+
+✅ When we talk about "Less is More", we are talking about simplifying the way we present ideas to focus the user on making decisions about a task in the most effective way possible.
+
+Here are a few helpful ways we can employ a “Less is more” approach:
+
+## Leave it out:
+
+"Just because we can does not mean we should"
+
+Including a design element, feature or product enhancement, because we have the technical ability to do so with no impact on the user experience. It could overcomplicate a simple task, cause cognitive overload and ultimately distract an audience from processing through a task with less friction.  It’s Equally as important to leave out a good idea than it is to include one, especially if it doesn’t help the user achieve their goals.
+
+![Figure: ❌ Bad example - overbranded background](microsoftteams-image-withlogos.png "Figure: Bad")
+
+![Figure: ✅ Good Example - reduced design clutter by limiting design elements](microsoftteams-image.png)
+
+Setting a bar or standard of minimum requirements for every new feature you would like to add would be a good way to measure and decide if it enhances the user experience and if it’s really necessary to add.
+
+## Keep it simple
+
+ By keeping things simple we reduce complexity and avoid the cognitive overload required for a user to make an informed decision.  We want to keep things easily digestible to as broad an audience as possible. Again mainly focus the user on the completion of a task. 
+
+Using simple descriptive language or a minimalist design approach can often demystify complicated topics and make information easily digestible so that users can make decisions with ease.
+
+![Figure: ❌ Bad example - Responding with no clear answer](responding-bad-example.jpg)
```

Comment:
Hey @2fernandez 

See how captions are done https://ssw.com.au/rules/rule/#7-captions
You won't need to add the emoji if you use the right markdown syntax

1. Please fix